### ISSUE TITLE
Requirment installation error. Error: pg_config executable not found 

### ISSUE THREAD

**Issue by alphaguy4** [Cloud-CV/EvalAI#1480]
on running 
`pip install -r requirements/dev.txt`

I am getting following error : 

 ```Using cached psycopg2-2.6.1.tar.gz
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    Error: pg_config executable not found.

    Please add the directory containing pg_config to the PATH
    or specify the full executable path with the option:

        python setup.py build_ext --pg-config /path/to/pg_config build ...

    or with the pg_config option in 'setup.cfg'.

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-U3FPuB/psycopg2/```
```

Environment 
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.1 LTS
Release:        16.04
Codename:       xenial

**Comment by alphaguy4** [comment]
We need to run 
`apt-get install libpq-dev`
before 
`pip install -r requirements/dev.txt`
Please update readme

**Comment by Obtin** [comment]
These are some of the problems that could be included in #1343. @deshraj.

**Comment by deshraj** [comment]
Closing since this is same as #1343
